devtools/pip-tools/pr_tool/backport.py

```diff
@@ -0,0 +1,357 @@
+#   BAREOS® - Backup Archiving REcovery Open Sourced
+#
+#   Copyright (C) 2023-2024 Bareos GmbH & Co. KG
+#
+#   This program is Free Software; you can redistribute it and/or
+#   modify it under the terms of version three of the GNU Affero General Public
+#   License as published by the Free Software Foundation and included
+#   in the file LICENSE.
+#
+#   This program is distributed in the hope that it will be useful, but
+#   WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+#   Affero General Public License for more details.
+#
+#   You should have received a copy of the GNU Affero General Public License
+#   along with this program; if not, write to the Free Software
+#   Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+#   02110-1301, USA.
+
+"""
+this module provides commands to do a backport of a specific
+bareos PR into another branch
+"""
+
+import logging
+from os import environ
+from sys import stdout, stderr
+from subprocess import run
+from tempfile import NamedTemporaryFile
+from importlib import resources
+from git import RemoteReference
+from git.exc import GitCommandError
+from .github import Gh, InvokationError, whoami
+
+logger = logging.getLogger(__name__)
```

Comment:
Wouldn't be __name__ expanded always to __main__ ? 
Maybe you want something like
```
from pathlib import Path
logging.getLogger( Path(__file__).stem )
``` 
Should create a backport logger.

Reply:
From what i understand from python, pr_tool/main.py will have __name__ set to  '__main__', since the script is started from python immediately. 
Then main.py does `from . import backport`, which will cause python to execute pr_tool/backport.py with __name__ set to `backport`.
So this should be correct, but i did not specifically test this!

Reply:
great thanks for the check